server/lists/index.js

```diff
@@ -47,6 +47,11 @@ module.exports = {
             endpoint: '/topics',
             type: listService.types.DYNAMIC,
         },
+        DCU_POLICY_TEAM_FOR_TOPIC: {
+            client: 'INFO',
+            endpoint: '/team/topic/stage/DCU_DTEN_INITIAL_DRAFT',
```

Comment:
What's your thinking here? Is this only needed for DTEN?

Reply:
its for all but i just needed to pickup a stage name. The way DCU are setup is that the topic only really have 2 teams:
one for draft one for sign off. The mappings are then duplicated for each flow type (DTEN, MIN and TRO). So it doesn't matter which '...INITIAL_DRAFT' stage we pick here as long as its one for DCU flow.